client/main.lua

```diff
@@ -869,7 +869,7 @@ CreateThread(function()
 								end
 							end
 
-							if displayText ~= "" then displayNUIText(displayText) end
+							if displayText ~= "" and closestDoor.data.hideLabel and closestDoor.data.hideLabel ~= true then displayNUIText(displayText) end --*
```

Comment:
```suggestion
							if displayText ~= "" and closestDoor.data.hideLabel ~= nil and not closestDoor.data.hideLabel then displayNUIText(displayText) end
```
This should be the right check, with this it first checks if the hideLabel option exists and then checks if it is not true with the `not` keyword